src/cli/main/cli.py

```diff
@@ -411,11 +412,32 @@ class Config(NamedTuple):
     cache: List[Dict[str, str]]
 
 
+def _get_named_temporary_file_name() -> str:
+    attempts = 0
+    file_name = ""
+    while attempts < 4 and file_name == "":
+        try:
+            with tempfile.NamedTemporaryFile(delete=True) as file:
+                file_name = file.name
+            sleep(1)
```

Comment:
Unnecessary sleep? I think the second one is enough

Reply:
Removing it